jobs/cjd_sample/src/test/java/SampleTests.java

```diff
@@ -0,0 +1,33 @@
+import com.google.samples.BatchOperationSample;
+import com.google.samples.CommuteSearchSample;
+import com.google.samples.CompanyAndJobCrudSample;
+import com.google.samples.CustomAttributeSample;
+import com.google.samples.EmailAlertSearchSample;
+import com.google.samples.HistogramSample;
+import com.google.samples.LocationSearchSample;
+import com.google.samples.SearchBasicSamples;
+import com.google.samples.SearchFeaturedJobsSample;
+import com.google.samples.SearchFiltersSample;
+import com.google.samples.SearchQuickstart;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public class SampleTests {
+
+  @Test
+  public void runAllSamples() throws Exception {
```

Comment:
So this test runs all the samples, but the only thing that would set it off is if one of them throws an exception.

The tests need to be a little more robust than that - verifying that the output of each sample methods match what is expected. You can do this by applying a new stream as System.out and then verifying its contents. Please see other tests in this repo for examples.

Reply:
Half done.